docs/guides/formulasC2/build-template.md

```diff
@@ -0,0 +1,260 @@
+4. Click **Save**.
+
+## Edit and Delete Variables
+
+You can edit the name of a variable or remove a variable from a formula template.
+
+{% include note.html content="You cannot delete a formula variable if an instance of the formula exists.  " %}
+
+To edit or delete a formula variable:
+
+1. Open the formula template. On the Formulas page, hover over the Formula Card, and then click Open.
+5. Click **Variables**.
+![Variables](img/variables.png)
+2. Click **Edit Variable**.
+3. Select the variable to edit or delete, and then:
+  * Edit the name, and then click **Save**. Or,
+  * Click **Delete**.
+
+## Review Executions
+
+You can troubleshoot formulas by reviewing executions. The executions are split into three columns from left to right:
+
+* Formula Executions &mdash; the list of executions of the selected formula template. Failed executions are clearly marked.
+* Steps &mdash;The steps in the formula and an indication of their success <img src="img/icon-step-success.png" alt="Success" class="inlineImage"> or failure <img src="img/icon-step-failure.png" alt="Failure" class="inlineImage">.
+* values &mdash;The details associated with the selected step.
+![Formula Instance Executions](img/instance-executions.png)
+
+{% include note.html content="To review executions, you must have a formula instance. Otherwise, the Executions button is unavailable.  " %}
+
+To review executions
+
+1. Open the formula template. On the Formulas page, hover over the Formula Card, and then click **Open**.
+2. Click **Executions**.
+![Variables](img/executions.png)
+3. Click the execution to review.
+4. Click the step to review.
+
+## Copy a Formula Template
+
+You can create a formula template based on an existing template or from a Cloud Element sample formula, resulting in a copy of the template.
+
+To create a copy of a formula template:
+
+1. Click **Formulas**, and then on the Formulas page, click **Create New Formula**.
+![New Formula Template](img/new-formula.png)
+3. Click **Create From Existing Formula**.
+  * To use one of your existing formula templates, select the template.
+  * To use a Cloud Elements formula template, click CE Sample Formulas, and then select the formula.
+4. Enter a name for your formula, and then click **Create**.
+
+
+## Parallel Executions
+
+To help formula instances execute the formula steps as efficiently as possible, formula instances take advantage of multithreaded executions. That is, multiple steps in a formula instance can make API calls at the same time. Some API providers do not allow multiple API calls from the same account at the same time. If this happens, you can change a formula template or an individual formula instance to execute one step at a time. This makes the formula less efficient and results in an increase in the overall execution time.
```

Comment:
This isn't accurate.  Steps within a given formula are not run in parallel, as that would not be possible since each step *usually* relies on step execution values from the previous.  Instead, multiple executions for a single formula instance *could* be run in parallel, for example if we get a bunch of events at once and that kicks off many formula instance executions.